e2e/tools/validator/validations.yaml

```diff
@@ -0,0 +1,136 @@
+validations:
```

Comment:
suggest to add "vm" in the name for clarification. 
can we add a field for query description? do not have to be in this PR anyway.

Reply:
I will think of a better way to do this (in a separate PR) since I have the idea of using the same for comparing 2 kepler running on the same machine.